db/migrate/20220508232038_add_columns_to_casa_orgs.rb

```diff
@@ -0,0 +1,7 @@
+class AddColumnsToCasaOrgs < ActiveRecord::Migration[7.0]
+  def change
+    add_column :casa_orgs, :twilio_account_sid, :string
```

Comment:
@harsohailB I'm wondering, but should these API credentials be hashed before storing them in the db? https://security.stackexchange.com/questions/180345/do-i-need-to-hash-or-encrypt-api-keys-before-storing-them-in-a-database

Maybe @compwron can give some suggestions.

Reply:
In general yes. Use https://guides.rubyonrails.org/active_record_encryption.html

Reply:
I will look into that and hash before storing!

Reply:
Encrypted the Twilio credentials. Added random key set for encryption in `config/credentials.yml.enc` locally, however, this is git ignored. This is why the tests are failing. @compwron How can we add such encryption key sets in other environments?

Reply:
Resolved by having a separate credential file for each environment.